test/__tests__/data/customerio_input.json

```diff
@@ -1,4 +1,26 @@
 [
+  {
+    "message": {
+      "channel": "web",
+      "type": "identify",
+      "userId": "cio_1234",
+      "integrations": {
+        "All": true
+      },
+      "traits": {
+        "email": "updated_email@example.com",
+        "id": "updated-id-value"
+      },
+      "sentAt": "2019-10-14T09:03:22.563Z"
+    },
+    "destination": {
+      "Config": {
+        "datacenterEU": false,
+        "siteID": "46be54768e7d49ab2628",
+        "apiKey": "19f0f734e3bb87a44596"
+      }
+    }
+  },
   {
```

Comment:
convert this test cases into one file (customerio.json)

Reply:
We are planning to upgrade this will take this up in that